x-pack/test/api_integration/apis/uptime/rest/telemetry_collectors.ts

```diff
@@ -134,6 +134,9 @@ export default function ({ getService }: FtrProviderContext) {
           min_length: 0,
         },
         fleet_no_of_unique_monitors: 0,
+        synthetics_service_browser_steps: [],
+        synthetics_service_no_of_tests: 0,
+        synthetics_service_enabled: false,
```

Comment:
nit: You could also test this by using [`getService('usageAPI').getTelemetryStats()`](https://github.com/elastic/kibana/blob/eea2fc333b89076b8c89b764b399aa5d233362d3/x-pack/test/api_integration/services/usage_api.ts#L41) 😇